Azure/sonic-buildimage
# [submodule]: Update sonic-sairedis pointer

Signed-off-by: Volodymyr Samotiy <volodymyrs@mellanox.com>



**- What I did**
Updated ```sonic-sairedis``` submodule pointer.

**- How I did it**
Updated ```sonic-sairedis``` submodule pointer.

**- How to verify it**
1. Deploy image to the switch and verify that all is up and running.
2. Run ansible tests.

**- Description for the changelog**
[submodule]: Update sonic-sairedis pointer
